### ISSUE TITLE
Bug with LET Imbibition capillary pressure

### ISSUE THREAD

**Issue by alifbe** [equinor/pyscal#45]
I noticed that there might be a bug for LET imbibition capillary pressure.

Here is the screenshot of the output from pyscal. As you see, I have set PcMin to -2 bar but the SWOF table generated pressure down to 2 bar.
 
![image](https://user-images.githubusercontent.com/11570927/67567348-066cee00-f72a-11e9-9fde-ff1af39a4455.png)

Propose fix:
```python
    def add_LET_pc_imb(self, Ls, Es, Ts, Lf, Ef, Tf, Pcmax, Pcmin, Pct):
        """Add an imbition LET capillary pressure curve.

        Docs: https://wiki.equinor.com/wiki/index.php/Res:The_LET_correlation_for_capillary_pressure
        """

        # Normalized water saturation including sorw
        self.table["swnpco"] = (self.table.sw - self.swirr) / (
                1 - self.sorw - self.swirr
        )

        # The "forced part"
        # ---- Original code
        # self.table["Fficow"] = (1 - self.table.swnpco) ** Ls / (
        #         (1 - self.table.swnpco) ** Ls + Es * self.table.swnpco ** Ts
        # )
        # ------------------
        self.table["Fficow"] = self.table.swnpco ** Lf / (
                self.table.swnpco ** Lf + Ef * (1 - self.table.swnpco) ** Tf
        )

        # The spontaneous part:
        # Original code:
        # self.table["Fticow"] = self.table.swnpco ** Lf / (
        #         self.table.swnpco ** Lf + Ef * (1 - self.table.swnpco) ** Tf
        # )
        # ------------------
        self.table["Fsicow"] = (1 - self.table.swnpco) ** Ls / (
                (1 - self.table.swnpco) ** Ls + Es * self.table.swnpco ** Ts
        )

        # Putting it together:
        # ---- Original code
        # self.table["pc"] = (
        #         (Pcmax + Pct) * self.table.Fficow - (Pcmin - Pct) * self.table.Fticow - Pct
        # )
        # ------------------
        self.table["pc"] = (
                (Pcmax - Pct) * self.table.Fsicow + (Pcmin - Pct) * self.table.Fficow + Pct
        )

        # Special handling of the interval [0,swirr]
        self.table.loc[self.table.swnpco < epsilon, "pc"] = Pcmax
        # and [1-sorw,1]
        # ---- Original code
        # self.table.loc[self.table.swnpco > 1 - epsilon, "pc"] = self.table.pc.min()
        # -------------------
        self.table.loc[self.table.swnpco > 1 - epsilon, "pc"] = Pcmin
        self.pccomment = (
                "-- LET correlation for imbibition Pc;\n"
                + "-- Ls=%g, Es=%g, Ts=%g, Lf=%g, Ef=%g, Tf=%g, Pcmax=%g, Pcmin=%g, Pct=%g\n"
                % (Ls, Es, Ts, Lf, Ef, Tf, Pcmax, Pcmin, Pct)
        )
```

**Comment by berland** [comment]
Thanks! Are you able to verify correctness of the PR #46 ?

**Comment by alifbe** [comment]
@berland Yes, I have tested with some different parameters and they look correct. Any idea when it will be in komodo?

**Comment by berland** [comment]
Merged, and released in 0.1.6. Komodo bleeding tomorrow perhaps.
